docker/serve/Dockerfile

```
@@ -24,7 +24,7 @@ RUN export FORCE_CUDA=1
 RUN pip install torchserve torch-model-archiver
 
 # MMLAB
-RUN pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
+RUN ["/bin/bash", "-c", "pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html"]
```

In fact, `CUDA` and `PYTORCH` variables cannot be used after `FROM`. Refers to [the docs](https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact).
We need to re-declare these args, like
```suggestion
ARG PYTORCH
ARG CUDA
RUN ["/bin/bash", "-c", "pip install mmcv-full==${MMCV} -f https://download.openmmlab.com/mmcv/dist/cu${CUDA//./}/torch${PYTORCH}/index.html"]
```